linter/linter.go

```diff
@@ -32,20 +34,26 @@ var Lints = []LintStruct{
 	},
 }
 
+type LinterInterface interface {
+	LintOCSPResp(*ocsp.Response)
+}
+
+type Linter struct{}
+
 // LintOCSPResp takes in a parsed OCSP response and prints its status
 // TODO: change function so that it returns a list of failed lints
-func LintOCSPResp(resp *ocsp.Response) {
-	fmt.Println("OCSP Response status: " + StatusIntMap[resp.Status]) // placeholder
+func (l Linter) LintOCSPResp(resp *ocsp.Response) {
```

Comment:
what does this syntax do?

Reply:
It means that `LintOCSPResp` is to be called from the Linter struct (so instead of `LintOCSPResp(resp)`, you need to do `l := Linter{}, l.LintOCSPResp(resp)`.